docs/technical-setup/install-a-git-client.md

```diff
@@ -15,14 +15,60 @@ It is therefore a user-friendly option which does not require you to use the com
 
 GitKraken is _free_ to use with public repositories like `OpenScPCA-analysis`; you do not need to upgrade to their paid plan!
 
-### Install GitKraken
+### Install GitKraken on macOS
 
-1. Install GitKraken [using this link](https://www.gitkraken.com/download). 
+1. Install GitKraken [using this link](https://www.gitkraken.com/download).
 Note that installing GitKraken will also provide you with Git itself.
 
 1. Set up GitKraken on your machine by [directly signing in with your GitHub account](https://help.gitkraken.com/gitkraken-client/github-gitkraken-client/#sign-in-with-github).
 This will automatically provide you with the credentials you need to interact with GitHub without further setup.
 
+### Install GitKraken on Windows with WSL2
+
+!!! note
+    Before installing GitKraken, make sure your WSL2 installation is up to date by running this command in Windows Powershell (not in Ubuntu).
+    When launching Powershell, make sure you are [running as an administrator](https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7.4#with-administrative-privileges-run-as-administrator).
+
+    Still need to install WSL2?
+    [Follow these instructions first](./install-wsl2.md).
+
+You will [install GitKraken as a Linux application](https://help.gitkraken.com/gitkraken-client/how-to-install/#deb) into the WSL2 side of your computer.
+Note that installing GitKraken will also provide you with Git itself.
+
+All of the following installation steps are commands that you should run in the Ubuntu app.
+
+1. To begin, open the Ubuntu app and run this command:
+
+    ```sh
+    sudo apt update
+    ```
+
+    This command ensures that `apt`, the native Ubuntu package manager, is up to date.
+    Note that Ubuntu will prompt you for your WSL2 password; again, no symbols will appear when you type it.
```

Comment:
I don't think this should be required? It won't really hurt, but it is usually an infrequently required command, as it really just makes sure the package index is up to date (not apt itself). We might want to put this command right at the end of the initial installation, and that should be sufficient for most cases.